### PR TITLE
More styled components

### PR DESCRIPTION
**Pull request by Caruki**
More adjustments with styled components

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
-import '../components/Buttons.css';
+import styled from '@emotion/styled';
 
-export default function Button(props) {
-  return <button className={props.className || 'button'} {...props} />;
-}
+const Button = styled.button`
+  background-color: rgba(164, 173, 235, 1);
+  border-radius: 35px;
+  width: ${props => (props.btnType === 'nav' ? '85%' : '60%')};
```
Comment (marwinburesch) [inline src/components/Button.js]: try not to use abbreviations like `btn`. I know it's all over the internets, but it could lead to stuff like `e` and `i` ...

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  justify-content: center;
+  min-height: 300px;
+  margin: 30px;
+  margin-bottom: 15px;
+  width: 80%;
+  outline: none;
+  font-family: 'Lato', sans-serif;
+`;
+
+const baseFormQuestion = css`
```
Comment (marwinburesch) [inline src/components/Form.js]: very nice idea

File: src/components/Form.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+export const FormResultAnswer = styled.div`
+  ${baseFormOutput}
+  width: 85%;
+  background-color: #3f4989;
+  color: #a4adeb;
+  border: none;
+  cursor: default;
+`;
+
+export const FormResultQuestion = FormVoteQuestion.withComponent('div');
```
Comment (marwinburesch) [inline src/components/Form.js]: 🧐

File: src/components/Form.js (hunk 2 above)
Comment (Caruki) [reply src/components/Form.js]: thanks to Leon 👍

File: src/App.js [hunk 4]
```diff
@@ -5,7 +5,9 @@ import Add from './pages/Add';
 import Vote from './pages/Vote';
 import Result from './pages/Result';
 import Home from './pages/Home';
+import Polls from './pages/Home';
```
Comment (lmachens) [inline src/App.js]: Shouldn't it be `import Polls from './pages/Polls';`?

File: src/App.js [hunk 5]
```diff
@@ -23,16 +25,20 @@ const Main = styled.main`
 function App() {
   return (
     <Router>
+      <GlobalStyles />
       <AppContainer>
         <AppHeader />
         <Main>
           <Switch>
-            <Route exact path="/polls/:pollId">
+            <Route exact path="/polls/:pollId/result">
```
Comment (lmachens) [inline src/App.js]: Why did you change the route to `/result`?

File: src/components/Button.js [hunk 6]
```diff
@@ -1,29 +1,24 @@
...
+  padding: ${props => (props.variant === 'nav' ? '20px' : '10px')};
   outline: none;
-  font-size: ${props => (props.btnType === 'nav' ? '1.1rem' : '0.8rem')};
+  font-size: ${props => (props.variant === 'nav' ? '1.1rem' : '0.8rem')};
   border: transparent;
   font-family: 'Lato', sans-serif;
   font-weight: 300;
   text-align: center;
-  margin-top: ${props => (props.btnType === 'nav' ? '20px' : '10px')};
+  margin-top: ${props => (props.variant === 'nav' ? '20px' : '10px')};
```
Comment (lmachens) [inline src/components/Button.js]: Alternative A:
`margin: ${props => (props.variant === 'nav' ? '20px' : '10px')} 10px 10px 10px;`

Alternative B:

```
margin: 10px;
margin-top: ${props => (props.variant === 'nav' ? '20px' : '10px')};
```

File: src/components/GlobalStyles.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function GlobalStyles() {
+  return (
+    <Global
+      styles={css`
+        body {
+          margin: 0;
+          background-image: linear-gradient(to bottom right, #303e69, #08153b);
+          /* height: 100vh; */
+          overflow: hidden;
```
Comment (lmachens) [inline src/components/GlobalStyles.js]: Why do you need `overflow: hidden`?

File: src/components/Form.js (hunk 2 above)
Comment (marwinburesch) [reply src/components/Form.js]: lol
